wordle-web/src/App.vue

```diff
@@ -1,86 +1,13 @@
 <template>
   <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
     <div class="wrapper">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">Class</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
+      <NavBar class="ma-10"></NavBar>
     </div>
   </header>
 
   <RouterView />
```

Comment:
You could potentially put something else on the home page so it isn't just the boring Vue documentation stuff.

Reply:
We plan on making changes in the future, we just didn't know what we wanted to do with it yet.